vicon/udp.h

```diff
@@ -57,12 +51,7 @@ class ObjectData
         m_FrameNumber = frameNumber;
 
         // Copy vectors into class
-        m_Position[0] = x;
-        m_Position[1] = y;
-        m_Position[2] = z;
-        m_Attitude[0] = yaw;
-        m_Attitude[1] = pitch;
-        m_Attitude[2] = roll;
+        m_Pose = std::move(pose);
```

Comment:
I don't think poses dynamically allocate anything that can be re-purposed so this probably doesn't do anything (the contents of ``pose`` will still have to be copied from the stack/registers to wherever ``m_Pose`` is located)

Reply:
Yeah that's true. We should just do ``m_Pose = pose;``